vars/piperPipelineStageInit.groovy

```diff
@@ -69,6 +69,15 @@ import static com.sap.piper.Prerequisites.checkScript
      * Example: `[$class: 'GitSCM', branches: [[name: <branch_to_be_cloned>]], userRemoteConfigs: [[credentialsId: <credential_to_access_repository>, url: <repository_url>]]]`.
      */
     'checkoutMap',
+    /**
+     * The map returned from a Jenkins git checkout. Used to set the git information in the
+     * common pipeline environment.
+     */
+    'scmInfo',
```

Comment:
We still want to pass this along to the setupCPE step.

Reply:
Yes, I understood. But we might be in trouble in case it is not provided and there is no checkout. See my other comment below. Maybe we should raise an exception rather than leaving it `null` in that case ...

Reply:
Done